coderd/coderd.go

```diff
@@ -391,6 +391,10 @@ func New(options *Options) *API {
 			r.Get("/resources", api.workspaceBuildResources)
 			r.Get("/state", api.workspaceBuildState)
 		})
+		r.Route("/entitlements", func(r chi.Router) {
+			r.Use(apiKeyMiddleware)
+			r.Get("/", entitlements)
```

Comment:
Just a question, it does not matter much at all.

Should all users be able to see available entitlements?

Reply:
Yes!

The Frontend calls this endpoint to decide what to show, so every logged in user needs access to the endpoint.